addons/web_tour/static/src/tour_service/tour_pointer_container.js

```diff
@@ -0,0 +1,11 @@
+/** @odoo-module **/
+
+import { Component, xml } from "@odoo/owl";
+
+export class TourPointerContainer extends Component {}
+TourPointerContainer.props = { pointers: Object };
+TourPointerContainer.template = xml`
```

Comment:
not sure why we need this. one pointer is enough, right? there is no need for a list of pointer?

Reply:
The good thing about it is that each tour will have it's own pointer controller and there will be no mixing of state. I tried removing it and I'm having difficulty to make it work properly.

I'd like to ask to keep this for the moment. Then when I find time, I'll try the single pointer. Will that be okay?